app/main.py

```diff
@@ -210,5 +212,12 @@ def encrypt_mock_data(mock_data: MutableMapping) -> str:
     return mock_data
 
 
+def get_schema_version(schema_folder, filename):
+    filepath = "/".join(["mock_data", schema_folder, filename])
+    with open(filepath) as f:
+        data = json.load(f)
+    return f'{data["schema_version"]}.0.0'
```

Comment:
Would this still need the suffix `.0.0` now?

Reply:
I don't believe so, I removed the suffix and the schemas seem to open correctly so I updated it 👍